app/code/Magento/Config/Block/System/Config/Edit.php

```diff
@@ -41,16 +44,24 @@ class Edit extends \Magento\Backend\Block\Widget
     protected $_configStructure;
 
     /**
+     * @var \Magento\Framework\Serialize\Serializer\Json
+     */
+    private $jsonSerializer;
+
+    /**
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Config\Model\Config\Structure $configStructure
      * @param array $data
+     * @param Json|null $jsonSerializer
      */
     public function __construct(
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Config\Model\Config\Structure $configStructure,
-        array $data = []
+        array $data = [],
+        Json $jsonSerializer = null
```

Comment:
it's better to have a dependency on the interface instead?
`Magento\Framework\Serialize\SerializerInterface`